common/static/common/js/discussion/views/discussion_thread_view.js

```diff
@@ -86,13 +86,6 @@
                     throw new Error("invalid mode: " + this.mode);
                 }
                 this.readOnly = $(".discussion-module").data('read-only');
-                this.model.collection.on("reset", function(collection) {
-                    var id;
-                    id = self.model.get("id");
-                    if (collection.get(id)) {
-                        self.model = collection.get(id);
-                    }
-                });
```

Comment:
@e-kolpakov  I'd rather not include code changes we don't understand, so please see if there is any difference with or without this change.

Thanks for stopping by, @Kelketek  :)